cmd/dagger/export.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+
+	"dagger.io/dagger"
+	"github.com/spf13/cobra"
+)
+
+var exportPath string
+
+var exportCmd = &FuncCommand{
+	Name:  "export",
+	Short: "Export an asset to the host (directory, file).",
+	OnSetFlags: func(cmd *cobra.Command, returnType *modTypeDef) error {
+		if returnType.Kind == dagger.Objectkind {
+			switch returnType.AsObject.Name {
+			case Directory, File:
```

Comment:
nit: `Container` has export too (writes tarball), think we may as well add it here

Reply:
Yes, I was just concerned with adding one or two examples on each root command and expand later but it's now quite easy to add a few more that make sense. 👍

Reply:
Done.